.github/workflows/4_builderpackage_manager.yml

```diff
@@ -191,10 +196,101 @@ jobs:
             echo "The installation could not be completed. The package will not be uploaded.";
             exit 1;
           fi
-
+          
+          echo "CONTAINER_ID=$CONTAINER_ID" >> $GITHUB_ENV
           echo "WAZUH_UID=$(cat $TESTS_PATH/wazuh_uid)" | tee -a $GITHUB_ENV
           echo "WAZUH_GID=$(cat $TESTS_PATH/wazuh_gid)" | tee -a $GITHUB_ENV
 
+      - name: Check installed files
+        uses: ./.github/actions/check_files
+        with:
+          expected_files: manager_base.csv
+          wazuh_uid: ${{ env.WAZUH_UID }}
+          wazuh_gid: ${{ env.WAZUH_GID }}
+          size_check: "false"
+          ignore: "site-packages,/var/ossec/framework/python"
+
+      - name: Test uninstall built manager
+        run: |
+          if [ -z "${{ env.PACKAGE_NAME }}" ] || [ -z "${{ env.CONTAINER_ID }}" ]; then 
+            echo "No package or container found!";
+            exit 1;
+          fi
+
+          sudo docker exec $CONTAINER_ID chmod +x /tests/uninstall_component.sh
+          sudo docker exec $CONTAINER_ID /tests/uninstall_component.sh $PACKAGE_NAME manager
+          sudo docker stop $CONTAINER_ID && sudo docker rm $CONTAINER_ID
+
+      - name: Get latest Wazuh release from GitHub
+        run: |
+          apt-get update && apt-get install -y curl jq
+          LATEST_TAG=$(curl -s https://api.github.com/repos/wazuh/wazuh/releases/latest | jq -r .tag_name | sed 's/^v//')
+          echo "Latest release detected: $LATEST_TAG"
+          echo "PUBLIC_RELEASE_VERSION=$LATEST_TAG" >> $GITHUB_ENV
+
+      - name: Download latest public wazuh-manager package
+        run: |
+          ARCH=${{ inputs.architecture }}
+          VERSION=${{ env.PUBLIC_RELEASE_VERSION }}
+
+          if [ "${{ inputs.system }}" = "deb" ]; then
+            FILE_NAME_PUBLIC="wazuh-manager_${VERSION}-1_${ARCH}.deb"
+            BASE_URL="https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-manager"
+          elif [ "${{ inputs.system }}" = "rpm" ]; then
+            FILE_NAME_PUBLIC="wazuh-manager-${VERSION}-1.${ARCH}.rpm"
+            BASE_URL="https://packages.wazuh.com/4.x/yum"
+          else
+            echo "Unsupported system type: ${{ inputs.system }}"
+            exit 1
+          fi
```

Comment:
I was wrong about the null chances of mixing arch-system during package name creation.

This can happen because workflow_dispatch architecture input does not sanitize their mixture, despite being functional. ie you can execute this workflow with deb and aarch64 and will fail at this step